core/src/test/java/com/linecorp/armeria/server/websocket/WebSocketServiceConfigTest.java

```diff
@@ -40,6 +40,7 @@ void webSocketServiceDefaultConfigValues() {
         assertThat(serviceConfig.requestAutoAbortDelayMillis()).isEqualTo(
                 WebSocketUtil.DEFAULT_REQUEST_AUTO_ABORT_DELAY_MILLIS);
 
+        // TODO: Should we allow default HttpServiceOptions to override the default values from virtualHostTemplate
```

Comment:
The priority of `HttpServiceOptions` is between the values in `ServiceBindingBuilder` and `VirtualHostBuilder`.

The new priority of configurations will be: (Listed from high priority to low priority)
- `ServiceBindingBuilder` (highest)
- `HttpServiceOptions` (if exists)
- `VirtualHostBuilder`
- `ServerBuilder` (lowest)

Based on the new priority, `sb.requestAutoAbortDelayMillis(1000)` should not override `HttpServiceOptions.requestAutoAbortDelayMillis()` if `requestAutoAbortDelayMillis > -1`.

Should we test two scenarios?
- Make sure that `sb.requestAutoAbortDelayMillis(1000)` does **NOT** override `serviceConfig.requestAutoAbortDelayMillis()` of `WebSocketService`
- Make sure that `sb.route().requestAutoAbortDelayMillis(1000)` overrides `WebSocketService.options().requestAutoAbortDelayMillis()`.

Reply:
Based on the new priority of configurations, I've added above mentioned test scenarios in `WebSocketServiceConfigTest#webSocketServiceOptionsPriority`.